bindings/object_store/src/lib.rs

```diff
@@ -196,6 +251,22 @@ fn format_object_store_error(err: opendal::Error, path: &str) -> object_store::E
     }
 }
 
+fn convert_entry(entry: &Entry, meta: &Metadata) -> ObjectMeta {
```

Comment:
**suggestion**: How about naming this function as `format_object_meta`?
**suggestion**: How about accepting `path: &str` isntead of an `Entry`?

Reply:
That looks good. Fixed!